SPECS/rpc/rpc.spec

```diff
@@ -46,6 +46,9 @@ cp LICENSE %{buildroot}%{_defaultlicensedir}/%{name}
 %license %{_defaultlicensedir}/%{name}/LICENSE
 
 %changelog
+* Wed Dec 3 2025 Polmoorx shiva kumar <polmoorx.shiva.kumar@intel.com> - 2.48.9-1
+- Upgraded the RPC component version from 2.45.1 to 2.48.9
```

Comment:
as part of practice, the changelog should indicate
1. package version upgrades
2. package bump is to fix CVE-2025-47913, CVE-2025-47914, CVE-2025-58181